### ISSUE TITLE
Broken with latest Sidekiq

### ISSUE THREAD

**Issue by dbussink** [iMacTia/sidekiq-logstash#30]
Looks like things are broken with the latest Sidekiq (6.5.0):

```json
{
  "ts":"2022-06-09T12:09:36.898Z",
  "pid":1,
  "tid":"187p",
  "lvl":"WARN",
  "msg":"NameError: uninitialized constant Sidekiq::ExceptionHandler\n\n config.error_handlers.delete_if { |h| h.is_a?(Sidekiq::ExceptionHandler::Logger) }"
}
```

Since https://github.com/mperham/sidekiq/pull/5340 this type no longer exists. I think this logic should be removed entirely since there's no default error handlers configured anymore from what I can see?

**Comment by iMacTia** [comment]
@dbussink I've fixed this in #31 and just released v2.0.2 in Rubygems
If you could give the new release a try and let me know if it fixed your issue, it would be greatly appreciated

**Comment by dbussink** [comment]
> @dbussink I've fixed this in #31 and just released v2.0.2 in Rubygems
> If you could give the new release a try and let me know if it fixed your issue, it would be greatly appreciated

Sorry for the delay, but looking all good!
